src/templates/comment-reply-input.jsx

```diff
@@ -286,10 +286,7 @@ const CommentReplyInput = createReactClass({
 
     if (this.state.isSubmitting) {
       submitLink = (
-        <div className="wpnc__spinner animated">
-          <span className="side left" />
-          <span className="side right" />
-        </div>
+        <Spinner className="wpnc__spinner" />
```

Comment:
Let's have `wpnc__spinner` be a default class

Reply:
Now that it's set as default, we no longer need to pass the class here?

Reply:
Yup we can remove this